network-api/networkapi/wagtailpages/models.py

```diff
@@ -560,6 +560,7 @@ def get_context(self, request):
         context = super().get_context(request)
         context = set_main_site_nav_information(self, context, 'Homepage')
         context = get_page_tree_information(self, context)
+        context['entries'] = self.get_children().live().order_by('-first_published_at')
```

Comment:
Why the hyphen before `first_published_at`?

Reply:
Newest first

Reply:
Yeah, but I'm asking is that just the syntax for how to write it here?...I looked at the docs and none of the fields had a hyphen before them.

Reply:
https://docs.djangoproject.com/en/2.2/ref/models/querysets/#order-by